util/Migrator/DbScripts_future/2023-06-FutureMigration.sql

```diff
@@ -0,0 +1,73 @@
+-- Remove Column
+IF COL_LENGTH('[dbo].[ApiKey]', 'ClientSecret') IS NOT NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[ApiKey]
+    DROP COLUMN
+        [ClientSecret]
+END
+GO
+
+-- Refresh views
+IF OBJECT_ID('[dbo].[ApiKeyDetailsView]') IS NOT NULL
+    BEGIN
+        EXECUTE sp_refreshview N'[dbo].[ApiKeyDetailsView]';
+    END
+GO
+
+IF OBJECT_ID('[dbo].[ApiKeyView]') IS NOT NULL
+    BEGIN
+        EXECUTE sp_refreshview N'[dbo].[ApiKeyView]';
+    END
+GO
+
+-- Drop existing SPROC
+IF OBJECT_ID('[dbo].[ApiKey_Create]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[ApiKey_Create]
+END
+GO
+
+-- Create the new SPROC
+CREATE PROCEDURE [dbo].[ApiKey_Create]
```

Comment:
`CREATE OR ALTER`

Reply:
resolved in https://github.com/bitwarden/server/pull/2943/commits/d3b2069012d03d9799ec5bd8087cd1e9633593bc